.node-version

```diff
@@ -1 +1,2 @@
-16.15.0
+18.13.0
+
```

Comment:
Is this file still going to be used? You called out in transcom/mymove#11053 that you were concerned about keeping this file around.

> It is, in my opinion, not a good idea to force everyone to reconfigure their version manager right now. But the change will be socialized so that folks can swap on their own schedules.